config/v1/types_oauth.go

```diff
@@ -555,6 +555,11 @@ type OpenIDClaims struct {
 	// If unspecified, no email is set for the identity
 	// +optional
 	Email []string `json:"email,omitempty"`
+
+	// groups is the list of claims value of which should be used to synchronize groups
+	// from the OIDC provider to OpenShift for the user
+	// +optional
+	Groups []string `json:"groups,omitempty"`
```

Comment:
is this a set? An ordered list? (you have to set list-type)

Reply:
This would be a set. How does one set a list-type? It does not seem to be used here yet.